inst/ms/aggreCAT.qmd

```diff
@@ -94,21 +126,21 @@ and aims to build automated tools that can rapidly and reliably assign
 "Confidence Scores" to research claims from empirical studies in the
 Social and Behavioural Sciences (SBS). Confidence Scores are
 quantitative measures of the likely reproducibility or replicability of
-a research claim or result, and may be used by consumers of Social and
-Behavioural Sciences research as a proxy measure for their credibility
-in the absence of replication effort.
+a research claim or result, and may be used by consumers of scientific
+research as a proxy measure for their credibility in the absence of
+replication effort.
 
 Replications are time-consuming and costly [@Isager2020], and studies
 have shown that replication outcomes can be reliably elicited from
 researchers [@Gordon2020]. Consequently, the DARPA SCORE program
```

Comment:
Should we be citing a SCORE paper as justifying why the SCORE paper is doing it's research? Is there at least another paper we can add?

Reply:
I thought the reference to Alipourfard several sentences above would cover that, but I've added it to directly follow the line you've highlighted.